packages/insured-bridge-relayer/src/LogHelper.ts

```diff
@@ -0,0 +1,44 @@
+import fs from "fs";
+import path from "path";
+import { Datastore } from "@google-cloud/datastore";
+const datastore = new Datastore();
```

Comment:
Do we not have to specify the project when creating this? How does it know which project's datastore to connect to?

Reply:
it's based off the provided GCP service account. as this will be running in prod it will access the main GCP service account. the database name is defined in the `datastore.key` call later on, with the DB called `RelayerUnprofitableLogs`